hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -24,12 +24,46 @@
         require: {
             overview: "^overview"
         },
-        controller: function ($http) {
+        controller: function ($http,$scope) {
             var ctrl = this;
+            $scope.reverse=true;
+            $scope.columnName="";
+            let nodeStatusCopy=[];
+            $scope.RecordsToDisplay ="10";
+            $scope.currentPage=1;
+            $scope.lastIndex = 0;
+
             $http.get("jmx?qry=Hadoop:service=SCMNodeManager,name=SCMNodeManagerInfo")
                 .then(function (result) {
                     ctrl.nodemanagermetrics = result.data.beans[0];
-                });
+                $scope.nodeStatus = ctrl.nodemanagermetrics && ctrl.nodemanagermetrics.NodeStatus && ctrl.nodemanagermetrics.NodeStatus.map(({ key, value}) =>{
```

Comment:
From this point on indentation seems to be shifted and a bit ad-hoc, please fix that, and also please go over the file, and fix missing spaces here and there (around = sign, after { and before }, after ,)

Reply:
Indentation has been fixed as well.